services/mailer/src/integration-tests/send-grid-transport.spec.ts

```diff
@@ -43,4 +46,9 @@ describe("SendGrid transport tests", () => {
 
     return request(app).post("/api/mailer/send").send(body).expect(CREATED);
   }).timeout(15000);
+
+  after(async () => {
+    const { container } = GLOBAL.bootstrap;
+    container.register("mailerConfig", asValue({ ...originalMailConfig }));
+  });
```

Comment:
Do we really need this?

Reply:
@eMKoded you have right we don't need it in this case. I have changed that.